apps/web/src/components/templates/email-editor/variables-management/VarItemsDropdown.tsx

```diff
@@ -1,12 +1,15 @@
 import { useState } from 'react';
-import { Collapse, UnstyledButton, useMantineTheme } from '@mantine/core';
+import { ActionIcon, Collapse, Tooltip, UnstyledButton, useMantineTheme } from '@mantine/core';
 import { ChevronUp } from '../../../../design-system/icons';
 import { ChevronDown } from '../../../../design-system/icons';
 import { VarItem } from './VarItem';
 import { colors } from '../../../../design-system';
+import { useClipboard } from '@mantine/hooks';
+import { Check, Copy } from '../../../../design-system/icons';
 
 export const VarItemsDropdown = ({ name, type }) => {
   const [open, setOpen] = useState(false);
+  const idClipboard = useClipboard({ timeout: 500 });
```

Comment:
small suggestion, you can destructure props from the `useClipboard` hook call like:
```
const { copied, copy } = useClipboard({ timeout: 500 });
```
and use them instead of `idClipboard.copied`...
to me it makes it easier to read, because at first I needed to think and check what actually the `idClipboard` is

Reply:
It should be okay 👍🏼 I've added a little bit of logic to have the right "var copied" to show to the user.
Feel free to give me feedbacks, thank you 🙏🏼